src/elfpy/markets.py

```diff
@@ -207,7 +285,7 @@ def tick(self, delta_time):
         """Increments the time member variable"""
         self.time += delta_time
 
-    def update_spot_price(self):
+    def update_spot_price_and_rate(self):
```

Comment:
can this live inside `update_market` instead of being its own function? In other words, are there times when we want to update the spot price & rate but do not want to update the other market variables?